### ISSUE TITLE
Support of INI files

### ISSUE THREAD

**Issue by rkarpuzov** [mikefarah/yq#1441]
Currently " -p p " looks similar syntax, but my tests failed with application.ini (firefox)"

Original file with:
````
#filter substitution
[App]
Vendor=MozillaTest
Name=LayoutDebug
Version=0.1
BuildID=@BUILD_ID@
Copyright=Copyright (c) 2004 Mozilla.org
ID={da915c15-c21a-41e2-95c3-2e0f76fd3191}
```

After change with  yq -i -p p '.Name = "Rossen"' application.ini
result is:

````

'[App]': ""
Vendor: MozillaTest
Name: Rossen
Version: "0.1"
BuildID: '@BUILD_ID@'
Copyright: Copyright (c) 2004 Mozilla.org
ID: '{da915c15-c21a-41e2-95c3-2e0f76fd3191}'

```'

After send run of the same command I receive more quotes in the result.

Best regards,
Rossen

**Comment by mikefarah** [comment]
yeah nice idea - I'm working on toml support at present, after I get that working I may work on this. Happy to receive PRs of course :)

**Comment by cz172638** [comment]
any update on it?
for me it looks similar to toml and properties

**Comment by Zash** [comment]
This can be closed now?

**Comment by rkarpuzov** [comment]
Yes!

In the test scenario the comment line is missing after processing, but this is subject of other discussion to keep the comments or no. 
INI support works for me and this is great job! 
Thank you all guys, who release this for me and the community!
